pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -18,6 +18,9 @@ The initial release of Windows Exporter Metrics contains a single collector avai
 | Key             | Description                                                            | Default   |
 | --------------- | ---------------------------------------------------------------------- | --------- |
 | scrape\_interval | The rate at which metrics are collected from the host operating system | 5 seconds |
+| we.logical\_disk.allow\_disk\_regex | Specify to be allowed regex for logical disk metrics                   | "/.+/"    |
+| we.logical\_disk.deny\_disk\_regex  | Specify to be denied regex for logical disk metrics                    | `NULL`    |
```

Comment:
```suggestion
| we.logical\_disk.deny\_disk\_regex  | Specify the regex for logical disk metrics to prevent collection of/ignore. Allow all by default. | `NULL`    |
```